text/Cargo.toml

```diff
@@ -15,7 +15,7 @@ diff = "0.1"
 regex.workspace = true
 chrono.workspace = true
 libc.workspace = true
-inotify = "0.10.2"
+notify-debouncer-full = "0.3.1"
```

Comment:
What's the benefit over simply `notify` crate?

Reply:
- Only emits a single Rename event if the rename From and To events can be matched
- Merges multiple Rename events
- Takes Rename events into account and updates paths for events that occurred before the rename event, but which haven’t been emitted, yet
- Optionally keeps track of the file system IDs all files and stiches rename events together (FSevents, Windows)
- Emits only one Remove event when deleting a directory (inotify)
- Doesn’t emit duplicate create events
-Doesn’t emit Modify events after a Create event